### PR TITLE
Update contributor badge

### LINKED ISSUE #459: Add additional badges like those in Bokeh
image

### PR DESCRIPTION
**Pull request by thuydotm**
Fixes #459, #738 

## Proposed Changes

  - added Contributors badge
  - added Static Analysis badge for python
  - reorganized badges into a table
  - dropped Contributor list in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -285,24 +285,6 @@ However, wrapping GDAL has a few drawbacks for Python developers and data scient
...
-- @rsignell-usgs
-- @marcozimmermannpm
-- @jthetzel
-- @chase-dwelle
-- @SAN154
-- @SapirLastimoza-Dooley
-- @lex-c
-
-Cite our code:
+**Cite our code:**
```
Comment (tcmetzger) [inline README.md:288]: Shouldn't this be a headline?

`#### Cite our code`

File: README.md [hunk 2]
```diff
@@ -222,24 +285,6 @@ However, wrapping GDAL has a few drawbacks for Python developers and data scient
...
```
Comment (tcmetzger) [inline README.md:288]: Shouldn't this be a headline?

`#### Cite our code`

File: README.md [hunk 3]
```diff
@@ -1,12 +1,75 @@
...
+  <td>Static Analysis</td>
+  <td>
+    <div>
+      <a href="https:https://codecov.io/gh/makepath/xarray-spatial">
+      <img alt="Language grade: JavaScript" src="https://codecov.io/gh/makepath/xarray-spatial/branch/master/graph/badge.svg"/>
+      </a>
+    </div>
+    <div>
+      <a href="https://lgtm.com/projects/g/makepath/xarray-spatial/context:python">
+      <img alt="Language grade: Python" src="https://img.shields.io/lgtm/grade/python/g/makepath/xarray-spatial.svg?"/>
```
Comment (tcmetzger) [inline README.md:65]: Why does this show up as "invalid" in the rendered version of the readme? 
image

Unless there is an easy fix, how about we leave this shield out for now?

**Review by tcmetzger — CHANGES_REQUESTED** [review]
Thank you, I have 2 comments/questions!

File: README.md (hunk 2 above)
Comment (thuydotm) [reply README.md]: Thanks Timo! Changed as requested.

File: README.md (hunk 3 above)
Comment (thuydotm) [reply README.md]: Sounds good, I updated the file to exclude this invalid badge.

**Review by tcmetzger — APPROVED** [review]

**Comment by tcmetzger** [comment]
Thank you!

**Pull request merged**